styles.css

```diff
@@ -424,3 +431,303 @@ footer hr {
   margin-top: 1vh;
   padding-top: 0%;
 }
+
+@media (min-width: 768px) {
+  .top {
+    display: none;
+  }
+
+  .logo {
+    color: #fff5e1;
+    font-family: 'Inter', serif;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    margin-left: 4%;
+  }
+
+  #navigation ul {
+    display: flex;
+  }
+
+  #navigation ul a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  #skills-ul li {
+    padding-left: 0%;
+  }
+
+  #navigation ul li {
+    padding-right: 9%;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    font-family: 'Inter', serif;
+  }
+
+  #desktop-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .hero {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    height: 100%;
+    padding-top: 9%;
+    padding-bottom: 9%;
+    background-image: url("Images/background-desktop.png");
+    background-position: right center;
+    background-size: 50% 100%;
+    background-repeat: no-repeat;
+  }
+
+  .hero p {
+    width: 50%;
+  }
+
+  #social-icons {
+    margin-right: 7%;
+    display: flex;
+    align-items: center;
+  }
+
+  #social-icons ul {
+    display: flex;
+    flex-direction: column;
+  }
+
+  #hero-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .description {
+    width: 60%;
+  }
+
+  .card-body-head p {
+    margin-top: 0%;
+  }
```

Comment:
- [ ] Your regular projects should display a 'see project' button on hover. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/171907384-f91191a0-0fd3-40d8-9846-20e980347cc8.png)